src/js/components/navigation.js

```diff
@@ -129,6 +137,18 @@ navigation = behavior(
         }
       },
     },
+    keydown: {
+      [NAV_PRIMARY]: keymap({ Escape: hideActiveNavDropdown }),
```

Comment:
Should focus be managed explicitly when <kbd>Escape</kbd> is pressed when focused on one of the submenu items? Similar to how the modal works, and based on related guidance here:

https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html#kbd2_label

>`Escape`
>- Closes submenu.
>- **Moves focus to parent menubar item.**